recipes/base_install.rb

```diff
@@ -236,14 +236,5 @@
 # Install FSx options
 include_recipe "aws-parallelcluster::_lustre_install"
 
-# Install EFA
-if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon'
-  unless node['cfncluster']['cfn_region'].start_with?("cn-")
-    include_recipe "aws-parallelcluster::_efa_install"
-  else
-    package 'openmpi-devel' do
-      retries 3
-      retry_delay 5
-    end
-  end
-end
+# Fetch EFA Driver
+include_recipe "aws-parallelcluster::_efa_fetch"
```

Comment:
Better keep the check for OS and China

Reply:
the driver should be downloaded on all os's just not installed on os's that don't support. Added china check back